src/MongoDB/Query.c

```diff
@@ -228,6 +228,23 @@ HashTable *php_phongo_query_get_debug_info(zval *object, int *is_temp TSRMLS_DC)
 	ADD_ASSOC_LONG_EX(&retval, "limit", intern->limit);
 	ADD_ASSOC_LONG_EX(&retval, "batch_size", intern->batch_size);
 
+	if (intern->read_concern) {
+#if PHP_VERSION_ID >= 70000
+		zval read_concern;
+
+		php_phongo_read_concern_to_zval(&read_concern, intern->read_concern);
+		ADD_ASSOC_ZVAL_EX(&retval, "readConcern", &read_concern);
```

Comment:
Valgrind reported no issues. I poked into PHP 7's code for `add_assoc_zval_ex()` and it does appear to copy the zval argument. See: http://lxr.php.net/xref/PHP_7_0/Zend/zend_hash.c#552 and references to the `pData` argument therein. It's used with `ZVAL_COPY_VALUE()` when assigning the index.